src/main/java/com/tikal/hudson/plugins/notification/Protocol.java

```diff
@@ -78,7 +77,7 @@ protected void send(String url, byte[] data, int timeout, boolean isJson) throws
             connection.setRequestProperty("Content-Type", String.format( "application/%s;charset=UTF-8", isJson ? "json" : "xml" ));
             String userInfo = targetUrl.getUserInfo();
             if (null != userInfo) {
-              String b64UserInfo = DatatypeConverter.printBase64Binary(userInfo.getBytes());
+              String b64UserInfo = new String(userInfo.getBytes());
```

Comment:
This is definitely changing the behavior. 
You likely want to re-read this code, and read about https://en.wikipedia.org/wiki/Base64

Probably @mrinal1209 you want to test this on your IDE or so, and basically print to stdout for instance to understand the difference between the previous code, and the change you are proposing.